_data/navigation.yml

```diff
@@ -110,12 +110,6 @@ docs_integrate:
     children:
       - title: "API Documentation"
         url: /integrate/connector-api
-      - title: "Sending Messages"
```

Comment:
Ich fand das war in /integrate schon sehr gut aufgehoben, war ja die integrator demo für diese Szenarien.